Loop54.Tests.NetCore/Loop54.Tests.NetCore.csproj

```diff
@@ -25,6 +27,12 @@
     <ProjectReference Include="..\Loop54.NetStandard\Loop54.NetStandard.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <None Include="..\Loop54.Shared\sign2021.snk">
```

Comment:
Is there a reason to have the year in the filename? I'd rather emphasise that this "secret" key is not actually secret at all, but is open to the world. "Loop54.Connector.NotSecret.snk"? :)